validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json

```diff
@@ -0,0 +1,12 @@
+{
+  "contact": {
+    "email": "contact@cryptogenesis.info",
+    "telegram": "@crypt0genesis",
+    "twitter": "@Crypto0Genesis",
+    "discord": "Crypto🔗Genesis"
```

Comment:
Missing a comma on the Discord line, causing the file to not have proper JSON formatting.

Reply:
Hi, let me check and fixed it.

Reply:
> Missing a comma on the Discord line, causing the file to not have proper JSON formatting.

Fixed and I have updated the repository. Please check and let me know. Thank you!!

Reply:
Do I have to pull new request?